programs/fileio.c

```diff
@@ -1478,18 +1530,24 @@ FIO_compressFilename_internal(FIO_prefs_t* const prefs,
     /* Status */
     
     DISPLAYLEVEL(2, "\r%79s\r", "");
-    if (g_display_prefs.displayLevel > 2 || (g_display_prefs.displayLevel == 2 && prefs->nbFiles <= 1)) {
-        if (readsize == 0) {
-            DISPLAYLEVEL(2,"%-20s :  (%6llu => %6llu bytes, %s) \n",
-                srcFileName,
-                (unsigned long long)readsize, (unsigned long long) compressedfilesize,
-                dstFileName);
-        } else {
-            DISPLAYLEVEL(2,"%-20s :%6.2f%%   (%6llu => %6llu bytes, %s) \n",
-                srcFileName,
-                (double)compressedfilesize / readsize * 100,
-                (unsigned long long)readsize, (unsigned long long) compressedfilesize,
-                dstFileName);
+    if (g_display_prefs.displayLevel >= 2) {
+        if (g_display_prefs.displayLevel >= 3 || fCtx->nbFilesTotal <= 1) {
+            if (readsize == 0) {
+                DISPLAYLEVEL(2,"%-20s :  (%6llu => %6llu bytes, %s) \n",
+                    srcFileName,
+                    (unsigned long long)readsize, (unsigned long long) compressedfilesize,
+                    dstFileName);
+            } else {
+                DISPLAYLEVEL(2,"%-20s :%6.2f%%   (%6llu => %6llu bytes, %s) \n",
+                    srcFileName,
+                    (double)compressedfilesize / readsize * 100,
+                    (unsigned long long)readsize, (unsigned long long) compressedfilesize,
+                    dstFileName);
+            }
+        }
+        if (fCtx->nbFilesTotal > 1) {
+            fCtx->totalBytesInput += (size_t)readsize;
+            fCtx->totalBytesOutput += (size_t)compressedfilesize;
```

Comment:
I think it would make sense to always keep track of `totalBytesInput` and `totalBytesOutput`. Even if `g_display_prefs.displayLevel < 2`. Future readers will probably expect that these fields are always accurate.